traffic_monitor/towrap/towrap.go

```diff
@@ -277,8 +312,30 @@ func (s TrafficOpsSessionThreadsafe) trafficMonitorConfigMapRaw(cdn string) (*tc
 	if ss == nil {
 		return nil, ErrNilSession
 	}
-	configMap, _, error := ss.GetTrafficMonitorConfigMap(cdn)
-	return configMap, error
+	var configMap *tc.TrafficMonitorConfigMap
+	var err error
+
+	configMap, _, err = ss.GetTrafficMonitorConfigMap(cdn)
+	if configMap != nil {
+		json := jsoniter.ConfigFastest
+		data, err := json.Marshal(*configMap)
```

Comment:
This error needs logged or returned. As-is, it's writing the file if there was no error, but ignoring it if there is. We need to at least log it, and probably return it.